src/btree/bt_cursor.c

```diff
@@ -465,8 +465,9 @@ __wt_btcur_search_uncommitted(WT_CURSOR_BTREE *cbt, WT_UPDATE **updp)
 {
     WT_BTREE *btree;
     WT_CURSOR *cursor;
+    WT_DECL_RET;
     WT_SESSION_IMPL *session;
-    WT_UPDATE *upd;
+    WT_UPDATE *upd, *tmp_upd;
```

Comment:
Can alphabetize the variable declarations

Reply:
fd82c8e